packages/shared/providers.yaml

```diff
@@ -795,9 +795,17 @@ pipedrive:
     auth_mode: OAUTH2
     authorization_url: https://oauth.pipedrive.com/oauth/authorize
     token_url: https://oauth.pipedrive.com/oauth/token
+    token_response_metadata:
+        - api_domain
     proxy:
-        base_url: https://${connectionConfig.subdomain}.pipedrive.com
+        base_url: ${connectionConfig.api_domain}/api
         decompress: true
+        paginate:
```

Comment:
It might be dangerous to override an existing pagination system, in case other people are relying on it. 

@khaliqgant you added the previous pagination system (based on git blame). Maybe Pipedrive has different pagination per endpoint?

I'm starting to have second thoughts about having the pagination defined at the API level. Maybe we should keep it fully in scripts and consider it's an endpoint-level config.